docs/configurations.md

```diff
@@ -1,53 +1,70 @@
 # ScalarDB Configurations
 
-This document describes the configurations for ScalarDB.
+This page describes the available configurations for ScalarDB.
 
-## Transaction manager configurations
+## Transaction managers
 
-ScalarDB has several transaction manager implementations, such as Consensus Commit, gRPC, and JDBC, and you can specify one of the implementations with the `scalar.db.transaction_manager` property.
-This section describes the configurations for each transaction manager.
+Implemented within ScalarDB are three transaction managers: Consensus Commit, gRPC, and JDBC. You can specify one of the transaction managers by using the `scalar.db.transaction_manager` property and then specify the configurations for the transaction manager.
```

Comment:
@brfrn169 @josh-wong 
So, gRPC (ScalarDB Server) should probably be Cluster?
(or simply delete it since it's deprecated?)

Also, as we roughly discussed before, stating the JDBC transaction manager might be confusing to users since we don't recommend it.
Should we just delete it?

Reply:
I agree, we should delete the ScalarDB Server and the JDBC transaction manager. Additionally, they are mentioned in other documents, which should also be deleted.

> So, gRPC (ScalarDB Server) should probably be Cluster?

Yes, I think so. However, since Cluster is a feature of the enterprise edition, I'm wondering how to mention it.